test/common/crudtester.go

```diff
@@ -158,15 +163,14 @@ func (c *FederatedTypeCrudTester) CheckUpdate(template, placement, override pkgr
 	qualifiedName := federatedtypes.NewQualifiedName(template)
 
 	adapter := c.adapter.Template()
-	kind := adapter.Kind()
 
 	var initialAnnotation string
 	meta := adapter.ObjectMeta(template)
 	if meta.Annotations != nil {
 		initialAnnotation = meta.Annotations[AnnotationTestFederationCrudUpdate]
 	}
 
-	c.tl.Logf("Updating %s %q", kind, qualifiedName)
+	c.tl.Logf("Updating %s %q", c.kind, qualifiedName)
```

Comment:
(No action required) Again, from a review perspective, this kind of change would ideally be separated out.

Reply:
I definitely understand and mostly agree on your desire to have these things separated out from a review perspective. But it's too tedious to go back and undo each non-functional change from each individual commit, especially when a lot of this code was written during a mostly prototype phase. Moving forward, you can expect these things to be separated out.